### PR TITLE
Add Checkmark functionality

### PR DESCRIPTION
**Pull request by Ol-create**
- Create checkmark event method
- Add undo it by unchecking the checkbox
- Add updated tasks list to stored in local storage.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import './style.css';
+import loadEvents from './modules/loadevents.js';
```
Comment (FrederickMih) [inline src/index.js:2]: - [ ] Nice job for importing a new JavaScript file as a module. For us to fulfill the requirements of this project, the newly imported file should contain methods related to the status updates (`completed` : `true` / `false`). However, this file does not contain methods related to the status update.  This file is basically representing nothing for now. Please make sure to fix this issue and move a step forward towards approval.

File: src/modules/loadevents.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import * as myEvents from './eventListeners.js';
```
Comment (FrederickMih) [inline src/modules/loadevents.js:1]: - [ ] eventListeners.js file is currently empty. You will require a checkbox and an event listener on this checkbox. Please let's implement this. 
- [ ] Also, from the requirements of this project, the task object's values for `completed` are updated upon user actions( ie clicking on the checkbox)

File: src/index.html [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+            <a>
+                <span class="material-icons-outlined add">subdirectory_arrow_left</span>
+            </a>
+        </div>
+        <small id="warning" class="hide"></small>
+        <div class="listContainer">
+            <ul id ="list">
+                <!-- add dynamic list here... -->
+            </ul>
+            <a class="clearAll" href="#">Clear all completed</a>
```
Comment (FrederickMih) [inline src/index.html:29]: - [ ] In order to move a step forward towards the approval of this project, you will need a function for the `Clear all completed button` (This should be a button and not a link) using array.f filter() method. Please make sure to refer back to the requirement of this project. 
- [ ] At the end, make sure that the updated array of items is stored in the local storage, hence you will need an array of items in a list. Unfortunately, you haven't yet created any list. Please be sure to meet the requirement of this project and submit it once more

   Wish you all the best :pray:

**Review by FrederickMih — CHANGES_REQUESTED** [review]
Hi @Ol-create  , 

You have done a great job so far. Keep up! :rocket: 
There are some issues that you still need to work on to go to the next project but you are almost there!


# To Highlights


- Linters are correctly set up :heavy_check_mark: 
- Pull Request has a title and summary :heavy_check_mark: 
- README.md file is descriptive :heavy_check_mark: 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/eventListeners.js [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+export const deleteActivity = list.addEventListener('click', (e) => {
+  if (e.target.tagName === 'P') {
+    e.path[1].childNodes[5].classList.add('hide');
+    e.path[1].childNodes[7].classList.remove('hide');
+  } else if (e.target.innerHTML === 'delete') {
+    const value = e.path[2].childNodes[3].innerHTML;
+    tasks.eliminate(value);
+  }
+});
```
Comment (rustam-tolipov) [inline src/modules/eventListeners.js:34]: **You made a great effort on this project! It's working perfectly fine! 👏👏👏**

However, you also did the next project's activities such as delete action... In that case, you must put a link to the additional code in the Pull Request created for the next project. For example, you should create a link(s) to specific commit(s) and/or line(s) of code in Pull Request for Milestone 3 and include that link in the description of Pull Request for Milestone 4.

**Review by rustam-tolipov — APPROVED** [review]
## Hello @Ol-create 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉, 

Cheers and Happy coding!👏👏👏

Developer-512px-10

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**